README.md

```diff
@@ -58,35 +58,44 @@ pip install similarity-processor
 ### Commandline
 
 ```sh
->>>python -m similarity_processor.similarity_cmd --p "path\to\TestBank.xlsx" --u 0 --c "1,2,3" --n 8
+>>>python -m similarity --p "path\to\TestBank.xlsx" --u 0 --c "1,2,3" --n 8
 ```
 
 - Help option can be found at,  
 
 ```sh
->>>python -m similarity_processor.similarity_cmd --h
+>>>python -m similarity --h
```

Comment:
Thank you for making this change. The call now looks (and reads) more meaningful.